test/config_thunks.tests.js

```diff
@@ -168,6 +162,20 @@ describe('ReduxEntityConfig - thunks', () => {
             expect(dispatchedActionTypes).toNotInclude('users_DESTROY_FAILURE');
           });
       });
+
+      it('removes the entity from state', () => {
+        const promise = config.actions.destroy({ id: userStub.id });
+
+        return mockStore.dispatch(promise)
+          .then(() => {
+            const dispatchedActions = mockStore.getActions();
+            const destroySuccessAction = dispatchedActions.find(action => action.type === 'users_DESTROY_SUCCESS');
+            const state = mockStore.getState();
+
+            expect(destroySuccessAction.payload).toEqual({ data: { id: userStub.id } });
+            expect(config.reducer(state, destroySuccessAction)).toEqual(store);
```

Comment:
This is how to test the full reducer/thunk implementation using `redux-mock-store`. Since the mock store cannot be configured with reducers we need to explicitly call the reducer here.

More info here: https://github.com/arnaudbenard/redux-mock-store/issues/71#issuecomment-369546064